### ISSUE TITLE
Characterization of stratified cohort to show percentage based on the subgroup count

### ISSUE THREAD

**Issue by dimshitc** [OHDSI/Atlas#2840]
### Expected behavior
The percentage of covariates to be shown based on the number of patients in subgroup (so the user can see the pattern over time or age or any other stratification criteria used).

### Actual behavior
The percent is calculated based on the overall count of target cohort

### Implementation ideas
This issue depends on [#1876](https://github.com/OHDSI/WebAPI/issues/1876)
Since users are used to the current behavior , instead of changing it, would be nice to have a radio button, so users can chose how percentage is calculated
